recipe/meta.yaml

```diff
@@ -1,22 +1,32 @@
-{% set version = "0.9.12.post1" %}
+{% set version = "0.10.0" %}
 
 package:
   name: pythran
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/p/pythran/pythran-{{ version }}.tar.gz
-  sha256: e7589cf83b0befa9a1b55e98223caf89aff887d9e3f14be912cf8703a717f185
+  url: https://github.com/serge-sans-paille/pythran/archive/refs/tags/0.9.12.post1.tar.gz
+  sha256: 13333f8a13be4de677d49fd9d27a76c7256d262d4a7f7e8f716273be8980caf9
+  patches:
+    - patches/0001-0.10.0-palig.patch
 
 build:
-  number: 1
+  number: 0
   script:
     # prevent d1trimfile option being added which clang-cl does not understand.
     # d1trimfile option is added by a patch and disabled by unsetting SRC_DIR
     # https://github.com/conda-forge/python-feedstock/blob/e7ea437a819f8f06dcbc3769f6e46dc026070c44/recipe/patches/0021-Add-d1trimfile-SRC_DIR-to-make-pdbs-more-relocatable.patch
     - set "SRC_DIR="   # [win]
-    - python -c "import os, shutil; shutil.rmtree(os.path.join('pythran', 'xsimd'))"
-    - "python -c \"with open('setup.cfg', 'w') as s: s.write('[build_py]\\nno_xsimd=True')\""  # [not win]
+    # unvendor xsimd; use version packaged by conda-forge
+    - python -c "import os, shutil; shutil.rmtree(os.path.join('third_party', 'xsimd'))"
+    - "python -c \"with open('setup.cfg', 'w') as s: s.write('[build_py]\\nno_xsimd=True\\n')\""
+    # on windows, pythran wrongly sets %PREFIX\include instead of %PREFIX%\Library\include == %LIBRARY_PREFIX%\include;
+    # since this is executed through batch, dealing with '%' is horrible, especially because also cfg-parsing
+    # (see docs.python.org/3/library/configparser.html) would fail on bare % even if we manage to write correctly;
+    # avoid all the absurd consumption rules of '%' in batch by simply doing % -> chr(37)
+    - "python -c \"with open('setup.cfg', 'a') as s: s.write(f'[compiler]\\ninclude_dirs={chr(37)*2}LIBRARY_PREFIX{chr(37)*2}\\include')\""  # [win]
```

Comment:
Why are you trying to have `%` in the file? We should expand the env variable before writing to the file

Reply:
Disagree. Writing `%LIBRARY_PREFIX%` turns into `%PREFIX%\Library` turns into `D:\bld\...` which gets written as `D:ld\...` (because python interprets `\b`).

In other words, all the same escaping problems remain for writing the expanded variable, and after lots of futile tries, I'm pretty sure the `chr(37)` solution is the most elegant one (including legibility and reasonable length).

Reply:
Though, on second though, I'll try again with `r'...'` and see what comes out.

Reply:
... which - in turn - makes it hard to write line breaks into the cfg... All a bit tricky (without direct patching, which I'd like to avoid, however) 🙃

But open for ideas (that work 😄)

Reply:
OK, I'd have a solution** but then the issue is (since changed in the code) - we're not just talking about `setup.cfg` anymore (which wasn't active/effective at test time), but about the `pythran.cfg` that'll be part of the distributed artefact. So I think we have no choice but to write with `%`.

**
```diff
     # on windows, pythran wrongly sets %PREFIX\include instead of %PREFIX%\Library\include == %LIBRARY_PREFIX%\include;
-    # since this is executed through batch, dealing with '%' is horrible, especially because also cfg-parsing
-    # (see docs.python.org/3/library/configparser.html) would fail on bare % even if we manage to write correctly;
-    # avoid all the absurd consumption rules of '%' in batch by simply doing % -> chr(37)
-    - "python -c \"with open('pythran\\pythran.cfg', 'a') as s: s.write(f'\\n[compiler]\\ninclude_dirs={chr(37)*2}LIBRARY_PREFIX{chr(37)*2}\\include')\""  # [win]
+    # need to write line breaks '\n' as well as escape %LIBRARY_PREFIX% with r'...' -> separate strings
+    - "python -c \"with open('pythran\\pythran.cfg', 'a') as s: s.write('\\n[compiler]\\ninclude_dirs=' + r'%LIBRARY_PREFIX%\include')\""  # [win]
     # sanity check to ensure this is set correctly
     - ```

Reply:
> , but about the pythran.cfg that'll be part of the distributed artefact. So I think we have no choice but to write with %.

why not? conda will fix the prefix in the distributed artefact.

Reply:
Maybe I don't understand you correctly - I thought you meant writing the final value (something like `D:\bld\pythran_1631295533774\work`), which is obviously not portable.

Are you saying it should be
```
[compiler]
include_dirs=%%PREFIX%%\Library\include
```

Then I'm completely fine with that (but it still has to escape `%`)...

Reply:
> Maybe I don't understand you correctly - I thought you meant writing the final value (something like D:\bld\pythran_1631295533774\work), which is obviously not portable.

Yes, I meant writing the final value like `D:\bld\pythran_1631295533774\_h_env`.
Why is it obviously not portable?

Reply:
> Why is it obviously not portable?

Is this being caught by conda and replaced by `%PREFIX%` (in a random cfg-file)? Then I can believe it, I just didn't think of that.

Reply:
> in a random cfg-file

Conda would also have to be smart enough to write it as `%%PREFIX%%`, otherwise the cfg-parsing breaks...

Reply:
> Is this being caught by conda 

yes

> and replaced by %PREFIX% (in a random cfg-file)? Then I can believe it, I just didn't think of that.

No, it would be replaced by the correct value at install time in the user's machine.

Reply:
> No, it would be replaced by the correct value at install time in the user's machine.

That sounds very magical to me, but OK, cool.

Reply:
For some reason `%PREFIX%` doesn't seem to be set now...?

Despite inserting it without any escapes, the result of `type pythran\pythran.cfg` is:
```
[...snip....]

[compiler]
include_dirs=%PREFIX%\Library\include
```